packages/react-test-renderer/src/__tests__/ReactTestRendererAsync-test.js

```diff
@@ -158,7 +158,7 @@ describe('ReactTestRendererAsync', () => {
 
       expect(() =>
         expect(renderer).toFlushAndYieldThrough(['foo', 'baz']),
-      ).toThrow('Expected value to equal:');
+      ).toThrow('Difference:');
```

Comment:
The error matcher string changed, we no longer have the "Expected value to equal" text at the top of error outputs (shoutout to @pedrottimark)